tools/integration-tests/tester/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  tester:
+    image: iotaledger/goshimmer_tester
+    container_name: tester
+    build:
+      context: ../../..
+      dockerfile: Dockerfile-tester
```

Comment:
A bit tricky because you can't copy anything into an image outside of the Dockerfile. In combination with the reference to GoShimmer few directories up this is an issue. However, I solved it now with `image: golang:1.14` and made `tester` it's own module as well.